flink/src/main/scala/com/metamx/tranquility/flink/BeamSink.scala

```diff
@@ -32,8 +32,9 @@ import org.apache.flink.streaming.api.functions.sink.RichSinkFunction
   * This class provides a sink that can propagate any event type to Druid.
   *
   * @param beamFactory your implementation of [[BeamFactory]].
+  * @param throwOnDrop throws an exception if set to true and a message gets dropped.
   */
-class BeamSink[T](beamFactory: BeamFactory[T])
+class BeamSink[T](beamFactory: BeamFactory[T], throwOnDrop: Boolean = false)
```

Comment:
This is called `reportDropsAsExceptions` in other modules, could you use that name here?